docs/src/main/asciidoc/WildFly_Maven_Plugin_Guide.adoc

```diff
@@ -0,0 +1,410 @@
+some http://docs.wildfly.org/galleon/#_built_in_and_product_specific_options[options] that can be used at provisioning time 
+to provision server features at a specific stability level.
+
+The stability level handling described in the xref:Galleon_Guide.adoc#WildFly_Galleon_stability[Stability levels at provisioning time] 
+applies when provisioning a WildFly server.
+
+For example, provisioning a WildFly server that includes `preview` features and packages: 
+
+[source,xml]
+----
+<build>
+  <plugins>
+    <plugin>
+      <groupId>org.wildfly.plugins</groupId>
+      <artifactId>wildfly-maven-plugin</artifactId>
+      <configuration>
+        <feature-packs>
+          <feature-pack>
+            <location>wildfly@maven(org.jboss.universe:community-universe)</location>
+          </feature-pack>
+        </feature-packs>
+        <layers>
+          <layer>jaxrs-server</layer>
+        </layers>
+        <galleon-options>
+          <stability-level>preview</stability-level>
+        </galleon-options>
+      </configuration>
+      <executions>
+        <execution>
+          <goals>
+            <goal>package</goal>
+          </goals>
+        </execution>
+      </executions>
+    </plugin>
+  </plugins>
+</build>
+----
+
+[[wildfly-maven-plugin-additional-configuration]]
+== Additional configuration
+
+The plugin allows you to specify additional configuration items:
+
+* A set of WildFly CLI scripts to execute to fine tune the server configuration. 
+* Some extra content to be copied inside the server (e.g.: server keystore).
+
+[source,xml]
+----
+<build>
+  <plugins>
+    <plugin>
+      <groupId>org.wildfly.plugins</groupId>
+      <artifactId>wildfly-maven-plugin</artifactId>
+      <configuration>
+        ...
+        <packaging-scripts> (1)
+          <packaging-script>
+            <scripts>
+              <script>scripts/script1.cli</script>
+            </scripts>
+          </packaging-script>
+        </packaging-scripts>
+        <extra-server-content-dirs> (2)
+          <dir>extra-content/</dir>
+        <extra-server-content-dirs>
+      </configuration>
+      <executions>
+        <execution>
+          <goals>
+            <goal>package</goal>
+          </goals>
+        </execution>
+      </executions>
+    </plugin>
+  </plugins>
+</build>
+----
+
+(1) Execute the `scripts/script1.cli` script
+(2) Copy the content of the `extra-content/` directory to the root of the provisioned server.
+
+You can check https://docs.wildfly.org/wildfly-maven-plugin/[this documentation] 
+for more details on how to configure the WildFly Maven Plugin `package` goal and in particular how to 
+execute CLI scripts and add some extra content.
+
+[[wildfly-maven-plugin-build]]
+== Packaging your application
+
+Call ``` mvn package```  to provision a server and deploy your application. The 
+directory ```<project build directory>/server``` contains the WildFly installation.
```

Comment:
Maybe change to `${project.build.directory}/server`. It renders a little strange is the only reason I mention this.

Reply:
Thank-you. Done